OpenIGTLinkIF/MRML/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ if(OpenIGTLink_PROTOCOL_VERSION GREATER 1)
     vtkMRMLIGTLTrackingDataQueryNode.cxx
     vtkMRMLIGTLTrackingDataBundleNode.cxx
     vtkMRMLIGTLSensorNode.cxx
+    vtkSlicerOpenIGTLinkIFCommand.cxx
```

Comment:
It should be just vtkSlicerOpenIGTLinkCommand (without IF)

Reply:
Currently, the class in OpenIGTLinkIF is vtkSlicerOpenIGTLinkIFCommand, in order to avoid using the same name as the class in SlicerIGT.

Should the class in SlicerIGT be removed, and this one renamed to vtkSlicerOpenIGTLinkCommand?

Reply:
Yes, please remove the class in SlicerIGT.